README.md

```diff
@@ -135,9 +135,33 @@ https://github.com/angular-ui/ui-router/wiki/Frequently-Asked-Questions#how-to-c
 
 ## Configuring TranSMART Endpoint
 
-Endpoints are defined in src/app/config.json
+TranSMART Endpoints are defined in [`src/app/config.json`](https://github.com/thehyve/transmart-base-ui/blob/dev/src/app/config.json)
 
-To serve as endpoint for TranSMART Glowing Bear, TranSMART Backend need to support Cross Origin Resource Sharing (CORS) 
-so that its APIs can be consumed by Glowing Bear. Enabling CORS support can be done in several ways such as allowing 
-cross origin resource sharing on application server where TranSMART Backend is deployed or installing Grails CORS plugin 
-in tranSMART Backend (development purpose only). 
+```
+{
+    "<environtment label>": {
+        "IS_TESTING": <flag to indicate if environtment for testing or not; true/false>,        
+        "IDLE_IN_MINUTES": <idle time in minutes>,        
```

Comment:
maybe add " eg 5" so that it is clear it is just an integer and not for instance '5m' or a string '5'